build/yamls/base/conf/antrea-agent.conf

```diff
@@ -32,7 +32,8 @@ featureGates:
 #  Egress: true
 
 # Enable AntreaIPAM, which can allocate IP addresses from IPPools. AntreaIPAM is required by the
-# bridging mode and allocates IPs to Pods in bridging mode.
+# bridging mode and allocates IPs to Pods in bridging mode. It is also required by secondary network
+# network IPAM.
```

Comment:
> It is also required by secondary network IPAM

If I interpret this correctly, we should stop shipping the Whereabouts binary altogether in the Antrea Docker image, as AntreaIPAM is the _only_ supported option now?

Reply:
The current implementation supports only Multus managed network, not Antrea native secondary network. I plan to wait for the SR-IOV CI ready to add Antrea IPAM support for native secondary network.